### PR TITLE
cleanup: Add missing docstring

### PR DESCRIPTION
**Pull request by VijaySharma815**

### CODE REVIEW

**Review by solarw — APPROVED** [review]
lgtm

File: cosmpy/ibc/core/connection/rest_client.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def Connection(self, request: QueryConnectionRequest) -> QueryConnectionResponse
 
         :param request: QueryConnectionRequest
         :return: QueryConnectionResponse
-        """
+        """  # noqa: D401
```
Comment (5A11) [inline cosmpy/ibc/core/connection/rest_client.py:57]: I don't see any `raises` missing.

Comment (5A11) [reply cosmpy/ibc/core/connection/rest_client.py:57]: Similarly below

File: cosmpy/staking/interface.py [hunk 2]
```diff
@@ -68,7 +68,7 @@ def Validators(self, request: QueryValidatorsRequest) -> QueryValidatorsResponse
     @abstractmethod
     def Validator(self, request: QueryValidatorRequest) -> QueryValidatorResponse:
         """
-        Validator queries validator info for given validator address.
+        Validate queries validator info for given validator address.
```
Comment (5A11) [inline cosmpy/staking/interface.py:71]:
```suggestion
        Query validator info for given validator address.
```

Comment (5A11) [reply cosmpy/staking/interface.py:71]: Similarly below

File: cosmpy/staking/rest_client.py [hunk 3]
```diff
@@ -70,7 +70,7 @@ def __init__(self, rest_api: RestClient) -> None:
 
     def Validators(self, request: QueryValidatorsRequest) -> QueryValidatorsResponse:
         """
-        Validators queries all validators that match the given status.
+        Validate query all validators that match the given status.
```
Comment (5A11) [inline cosmpy/staking/rest_client.py:73]:
```suggestion
        Query all validators that match the given status.
```

Comment (5A11) [reply cosmpy/staking/rest_client.py:73]: Similarly below.

File: scripts/check_copyright.py [hunk 4]
```diff
@@ -18,7 +18,7 @@
 #
 # ------------------------------------------------------------------------------
 """
-This script checks that all the Python files of the repository have:
+This script checks that all the Python files of the repository have:.
```
Comment (5A11) [inline scripts/check_copyright.py:21]: incorrect

File: cosmpy/ibc/core/connection/rest_client.py (hunk 1 above)
Comment (HashtrustTechnologies) [reply cosmpy/ibc/core/connection/rest_client.py:57]: D401: First line should be in imperative mood; try rephrasing

Comment (cmaliwal) [reply cosmpy/ibc/core/connection/rest_client.py:57]:
```
cosmpy/common/rest_client.py:127:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/common/utils.py:30:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/crypto/address.py:90:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/ibc/core/connection/rest_client.py:52:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/rest_client.py:64:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:42:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:51:1: D401 First line should be in imperative mood
```

I didn't find any strong solution for this.

Comment (VijaySharma815) [reply cosmpy/ibc/core/connection/rest_client.py:57]:
```
cosmpy/common/rest_client.py:127:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/common/utils.py:30:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/crypto/address.py:90:1: D401 First line should be in imperative mood; try rephrasing
cosmpy/ibc/core/connection/rest_client.py:52:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/rest_client.py:64:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:42:1: D401 First line should be in imperative mood
cosmpy/ibc/core/connection/interface.py:51:1: D401 First line should be in imperative mood
```

I didn't find any strong solution for this.

File: tests/unit/test_aerial/gas_test.py [hunk 5]
```diff
@@ -64,25 +64,25 @@ def test_table_gas_estimation(input_msgs, expected_gas_estimate):
...
         self, subspace: str, key: str  # pylint: disable=unused-argument
     ) -> Any:
-        """Set query params"""
+        """Set query params."""
         return {"max_gas": -1}
 
 
 @pytest.mark.parametrize(
-    "input_msgs,expected_gas_estimate",
+    "input_msgs,expected_gas_estimate.",
```
Comment (5A11) [inline tests/unit/test_aerial/gas_test.py:85]: incorrect

File: tests/unit/test_aerial/urls_test.py [hunk 6]
```diff
@@ -25,7 +25,7 @@
 
 
 @pytest.mark.parametrize(
-    "input_url,parse_result",
+    "input_url,parse_result.",
```
Comment (5A11) [inline tests/unit/test_aerial/urls_test.py:28]: incorrect

**Comment by VijaySharma815** [comment]
@5A11 All checks have been passed.

File: cosmpy/staking/interface.py [hunk 7]
```diff
@@ -79,7 +79,7 @@ def ValidatorDelegations(
         self, request: QueryValidatorDelegationsRequest
     ) -> QueryValidatorDelegationsResponse:
         """
-        ValidatorDelegations queries delegate info for given validator.
+        ValidatorDelegation queries delegate info for given validator.
```
Comment (5A11) [inline cosmpy/staking/interface.py:82]:
```suggestion
        Query delegate info for given validator.
```

File: cosmpy/staking/interface.py [hunk 8]
```diff
@@ -90,7 +90,7 @@ def ValidatorUnbondingDelegations(
         self, request: QueryValidatorUnbondingDelegationsRequest
     ) -> QueryValidatorUnbondingDelegationsResponse:
         """
-        ValidatorUnbondingDelegations queries unbonding delegations of a validator.
+        ValidatorUnbondingDelegation queries unbonding delegations of a validator.
```
Comment (5A11) [inline cosmpy/staking/interface.py:93]:
```suggestion
       Query unbonding delegations of a validator.
```

File: cosmpy/staking/interface.py [hunk 9]
```diff
@@ -99,7 +99,7 @@ def ValidatorUnbondingDelegations(
     @abstractmethod
     def Delegation(self, request: QueryDelegationRequest) -> QueryDelegationResponse:
         """
-        Delegation queries delegate info for given validator delegator pair.
+        Delegate queries delegate info for given validator delegator pair.
```
Comment (5A11) [inline cosmpy/staking/interface.py:102]:
```suggestion
        Query delegate info for given validator delegator pair.
```

File: cosmpy/staking/rest_client.py [hunk 10]
```diff
@@ -124,7 +124,7 @@ def ValidatorUnbondingDelegations(
 
     def Delegation(self, request: QueryDelegationRequest) -> QueryDelegationResponse:
         """
-        Delegation queries delegate info for given validator delegator pair.
+        Delegate query delegate info for given validator delegator pair.
```
Comment (5A11) [inline cosmpy/staking/rest_client.py:127]:
```suggestion
        Query delegate info for given validator delegator pair.
```

**Pull request merged**